src/Owin.Security.OpenIdConnect.Server/Events/ValidationEndpointContext.cs

```diff
@@ -14,7 +14,7 @@
 namespace Owin.Security.OpenIdConnect.Server {
     /// <summary>
     /// An event raised before the authorization server handles
-    /// the request made to the token validation endpoint.
+    /// the request made to the token introspection endpoint.
```

Comment:
My mistake, I missed a couple. I'll fix that.

However, I still think there should be an indication that the validation endpoint implements the introspection spec (with some modifications and additions) somehow since it's a standard, in case implementers of `IOpenIdConnectServerProvider` need to modify things - thus including `introspection (validation)` in the comments there. Should this be a discussion topic?